iModelJsNodeAddon/presentation/ECPresentationSerializer.cpp

```diff
@@ -1119,6 +1131,365 @@ rapidjson::Document IModelJsECPresentationSerializer::_AsJson(ContextR ctx, Node
     return json;
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+static Utf8String GetECValueTypeName(PrimitiveType type)
+    {
+    switch (type)
+        {
+        case PRIMITIVETYPE_Binary:
+            return "binary";
+        case PRIMITIVETYPE_Boolean:
+            return "boolean";
+        case PRIMITIVETYPE_DateTime:
+            return "dateTime";
+        case PRIMITIVETYPE_Double:
+            return "double";
+        case PRIMITIVETYPE_Integer:
+            return "int";
+        case PRIMITIVETYPE_Long:
+            return "long";
+        case PRIMITIVETYPE_Point2d:
+            return "point2d";
+        case PRIMITIVETYPE_Point3d:
+            return "point3d";
+        case PRIMITIVETYPE_String:
+            return "string";
+        default:
+            return "";
+        }
+    }
+
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+static int ParsePrimitiveType(Utf8String const& typeName)
+{
+    if (typeName.Equals("binary"))
+        return PRIMITIVETYPE_Binary;
+    else if (typeName.Equals("boolean"))
+        return PRIMITIVETYPE_Boolean;
+    else if (typeName.Equals("dateTime"))
+        return PRIMITIVETYPE_DateTime;
+    else if (typeName.Equals("double"))
+        return PRIMITIVETYPE_Double;
+    else if (typeName.Equals("int"))
+        return PRIMITIVETYPE_Integer;
+    else if (typeName.Equals("long"))
+        return PRIMITIVETYPE_Long;
+    else if (typeName.Equals("point2d"))
+        return PRIMITIVETYPE_Point2d;
+    else if (typeName.Equals("point3d"))
+        return PRIMITIVETYPE_Point3d;
+    else if (typeName.Equals("string"))
+        return PRIMITIVETYPE_String;
+    return 0;
+}
+
+/*---------------------------------------------------------------------------------**//**
+// @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+rapidjson::Document GetPoint2dJson(DPoint2dCR pt, rapidjson::MemoryPoolAllocator<>* allocator)
+    {
+    rapidjson::Document doc(allocator);
+    doc.SetObject();
+    doc.AddMember("x", pt.x, doc.GetAllocator());
+    doc.AddMember("y", pt.y, doc.GetAllocator());
+    return doc;
+    }
+
+/*---------------------------------------------------------------------------------**//**
+// @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+rapidjson::Document GetPoint3dJson(DPoint3dCR pt, rapidjson::MemoryPoolAllocator<>* allocator)
+    {
+    rapidjson::Document doc(allocator);
+    doc.SetObject();
+    doc.AddMember("x", pt.x, doc.GetAllocator());
+    doc.AddMember("y", pt.y, doc.GetAllocator());
+    doc.AddMember("z", pt.z, doc.GetAllocator());
+    return doc;
+    }

```

Comment:
should be `static`?

Reply:
applies to the functions below too

Reply:
Added `static`. Added to other functions as well, please check if I've missed any.